### PR TITLE
add instructions about Local Cluster E2E test by local-up-cluster.sh

### PR DESCRIPTION
**Pull request by hanxiaoshuai**
add instructions about Local Cluster E2E test by local-up-cluster.sh
ref issue [79](https://github.com/kubernetes-sigs/poseidon/issues/79)

### CODE REVIEW

File: docs/devel/README.md [hunk 1]
```diff
@@ -199,6 +199,9 @@ below script. It will build the release tar push it to docker locally and run th
 The ```'kubeconfig'``` path should be ```$HOME/.kube/config```.
 The below script run based on the above assumptions.
 And it should point to a running k8s cluster.
+If your running k8s cluster was started by local-up-cluster.sh, you should ```export HOSTNAME_OVERRIDE=$master-ip``` before running local-up-cluster.sh.
```
Comment (m1093782566) [inline docs/devel/README.md]: nit: If your running k8s cluster that is started by ...

File: docs/devel/README.md [hunk 2]
```diff
@@ -199,6 +199,9 @@ below script. It will build the release tar push it to docker locally and run th
 The ```'kubeconfig'``` path should be ```$HOME/.kube/config```.
 The below script run based on the above assumptions.
 And it should point to a running k8s cluster.
+If your running k8s cluster was started by local-up-cluster.sh, you should ```export HOSTNAME_OVERRIDE=$master-ip``` before running local-up-cluster.sh.
+```$master-ip``` is the IP of your machine where will run k8s cluster.
```
Comment (m1093782566) [inline docs/devel/README.md]: nit: ... is the non-loopback IP of your machine where running the k8s cluster.

**Comment by m1093782566** [comment]
Nice catch! LGTM except some nits.

File: docs/devel/README.md (hunk 1 above)
Comment (hanxiaoshuai) [reply docs/devel/README.md]: Done

File: docs/devel/README.md (hunk 2 above)
Comment (hanxiaoshuai) [reply docs/devel/README.md]: Done

**Pull request merged**